src/api-reference/user-provisioning/v4.user-provisioning.md

```diff
@@ -40,31 +78,34 @@ Name|Description|Endpoint
 `identity.user.core.read`|Read user core data.|GET
 `identity.user.coresensitive.read`|Read core sensitive data.|GET
 `identity.user.enterprise.read`|Read user enterprise data.|GET
-`travel.user.general.read`|Read general Travel data: Manager, Travel Name, RuleClass, Groups, OrgUnit, Travel Custom Fields|GET
-`travel.user.private.read`|Read private Travel data. XML Sync ID, CRSName/GDS Sync ID, Travel Name Remark, Gender|GET
+`travel.user.general.read`|Read general Travel data.|GET
+`travel.user.private.read`|Read private Travel data.|GET
 `spend.user.general.writeonly`|Change spend user information.|POST, PUT, PATCH
 `spend.user.general.read`|View spend user information.|GET
 
-## Dependencies <a name="dependencies"></a>
+## <a name="dependencies"></a>Dependencies
 
-SAP Concur users must purchase either Concur Expense or Concur Travel or both in order to use this API. This API only available to approved early adopter partners and clients. Please contact your SAP Concur representative for more information.
+SAP Concur users must purchase either Concur Expense or Concur Travel or both in order to use this API. This API is not available in the China Data Center.  Please contact your SAP Concur representative for more information.
 
-## Access Token Usage <a name="access-token-usage"></a>
+## <a name="access-token-usage"></a>Access Token Usage
 
 This API supports company level access tokens.
 
-## Events <a name="event-usage"></a>
+## <a name="event-usage"></a>Events
 
-UPS supports post event notification when provisioning process is complete.
+UPS supports post event notification when provisioning process is complete. Using the Event Subscription service will allows clients and partners to be notified through web services when certain actions take place in connected companies. 
 
-https://developer.concur.com/api-reference/callouts/post-event-notification.html
+https://developer.concur.com/api-reference/ess/v4.event-subscription.html
 
 Event Type:
 * `ProvisioningComplete`
 
-## Create a Provisioning Request for One or More Users <a name="create-bulk-post"></a>
 
-Creates one or more provisioning requests.
+## <a name="create-users-post"></a>Create a User Provisioning Request using POST with /Users endpoint
```

Comment:
We do not include REST specs in the title of an endpoint because it is included in the URI Templates. Please remove the "using POST" verbiage.